src/components/JokesCard.jsx

```diff
@@ -0,0 +1,20 @@
+import { 
+  Box
+} from "@chakra-ui/react";
+
+export default function JokesCard() {
+  return(
+    <Box 
+      as="strong" 
```

Comment:
Does it make sense to have the whole text wrapped in `<strong />` for every joke?